backend/accessibility_templates/aa-001.xml

```diff
@@ -0,0 +1,9 @@
+<AccessibilityAttributes>
+ <SubtitleAttributes>
+  <!-- WVTT -> contentType="text" codecs="wvtt" lang="en" -->
+  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:99"/>  <!--other-->
```

Comment:
If EBU-TT-D is provided for Live stream the carriage would be EBU-TT-D in MP4/ISOBMFF. This has the termID 3.